source/docs/guides/symfony.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Symfony Guide
+description: The Symfony Guide
+extends: _layouts.documentation
+section: content
+---
+
+# Symfony Guide
+
+Pest was built to be framework agnostic, but as it's easily extendable, it works well in [Symfony](https://symfony.com) applications.
+
+Now, in this guide, we are going to transform a Symfony test into a Pest test.
+
+A basic functionnal test in Symfony look like the following:
+
+```php
+<?php
+
+namespace App\Tests;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function testSomething()
+    {
+        $client = static::createClient();
+        $client->request('GET', '/');
+
+        $this->assertResponseIsSuccessful();
+        $this->assertSelectorTextContains('h1', 'Hello World');
+    }
+}
+```
+
+With Pest, the same test could be write like this:
+
+```php
+<?php
+
+uses(\Symfony\Bundle\FrameworkBundle\Test\WebTestCase::class);
```

Comment:
Can you start by creating a `Pest.php` with this content, and then talk about the test? I think it would make it cleaner.

Reply:
I can, but my idea was to introduce one test as single tests, then introduce the Pest power :muscle: 

Let me rethink it :wink: